### PR TITLE
Integrate New Features Into the New User Interface

### PR DESCRIPTION
**Pull request by Daramola98**
#### What does this PR do?
- Integrate Completed Features into new User Interface

#### Description of Task to be completed?
- Integrate New Gaming Screen
- Integrate New Landing Page
- Integrate User should be onboarded onto the application
- Integrate Sign Up and Login Pages
- Integrate Email Verification into the new user interface

#### How should this be manually tested?
- The following pivotal tracker stories below should be up and running

#### What are the relevant pivotal tracker stories?
#157801298, #157801299, #158121598, #157801294

### CODE REVIEW

File: public/js/controllers/index.js [hunk 1]
```diff
@@ -58,7 +58,8 @@ angular.module('mean.system').controller('IndexController', [
             $scope.user.profileImage = res.secure_url;
             $http.post('/api/auth/signup', $scope.user)
               .then(() => {
-                $location.path('/confirmaccount');
+                $('#signUpModal').modal('show');
+                // $location.path('/confirmaccount');
```
Comment (fortunee) [inline public/js/controllers/index.js]: Get rid of commented out code. @Daramola98

File: public/js/controllers/index.js [hunk 2]
```diff
@@ -71,12 +72,17 @@ angular.module('mean.system').controller('IndexController', [
         $http.post('/api/auth/signup', $scope.user)
           .then(
             () => {
-              $location.path('/confirmaccount');
+              // $location.path('/confirmaccount');
```
Comment (fortunee) [inline public/js/controllers/index.js]: Do same here and every other place we have commented out code @Daramola98

**Review by fortunee — COMMENTED** [review]
Hi, @Daramola98 Please check out my comments and clean up accordingly

File: public/js/controllers/index.js (hunk 2 above)
Comment (Daramola98) [reply public/js/controllers/index.js]: Done @fortunee

**Pull request merged**